packages/trpc/server/routers/viewer/webhook.tsx

```diff
@@ -19,60 +19,55 @@ const webhookIdAndEventTypeIdSchema = z.object({
   eventTypeId: z.number().optional(),
 });
 
-const webhookProcedure = authedProcedure.use(async ({ ctx, rawInput, next }) => {
-  // Endpoints that just read the logged in user's data - like 'list' don't necessary have any input
-  if (!rawInput) {
-    return next();
-  }
-  const webhookIdAndEventTypeId = webhookIdAndEventTypeIdSchema.safeParse(rawInput);
-  if (!webhookIdAndEventTypeId.success) {
-    throw new TRPCError({ code: "PARSE_ERROR" });
-  }
-  const { eventTypeId, id } = webhookIdAndEventTypeId.data;
+const webhookProcedure = authedProcedure
+  .input(webhookIdAndEventTypeIdSchema.optional())
+  .use(async ({ ctx, input, next }) => {
+    // Endpoints that just read the logged in user's data - like 'list' don't necessary have any input
+    if (!input) return next();
```

Comment:
here we also use multiple input parsers instead of `rawInput`.

Makes sure that if an input is passed, it follows the expected shape